### PR TITLE
Updated FontAwesome to 5.15.2 and made the changes needed

### PR DESCRIPTION
**Pull request by lbcnz**

### CODE REVIEW

**Comment by sylhare** [comment]
That's a lot of work on this one, I'll review it as well. Thanks!
How did you proceed to update the it to 5.15.2?

Also out of curiosity, do you know why the `fa` has been replaced by `fas`, `far`, `fab`? Maybe I should go read about the differences between v4 and v5.

**Comment by lbcnz** [comment]
They splitted the icons in different fonts: solid, regular and brands. FA 4 had 675 icons, FA 5.15.2 have 1609 free icons and 7865  pro icons.

About how I proceeded.  I did a similar update in December 2019 but did not made a PR at the time so I had an idea of what to do. I screened the theme files to understand how you implemented and the FA documentation to see what should divert from it or could be improved. I deleted the old FA `.scss` and fonts from the repo and copied the 5.15.2 in the place. As syntax slighted changed I had to replace `fa` accordingly by `[fas,far,fab]`. I used ripgrep to find the files that I had do edit and in some used vim/sed to search and replace the strings. Then tested to see if I missed something.

I though on using svg sprites instead of the fonts, as I did in the past but that would divert a lot more from what was currently being used and require an additional JS script to be loaded.

File: _includes/post_info.html [hunk 1]
```diff
@@ -7,9 +7,7 @@
       <img src="{{ author.avatar | relative_url }}">
     {% endif %}
     <p class="meta">
-      {% if author.name %}{{ author.name }} - {% endif %}
-      {% assign x = date | date: "%m" | minus: 1 %}
-      {{ site.data.language.str_months[x] }} {{ date | date: "%d, %Y" }}
+    {% if author.name %}{{ author.name }} - {% endif %} - {% endif %}{{ date | date: "%B %-d, %Y" }}
```
Comment (sylhare) [inline _includes/post_info.html:10]: Line 10 in `post_info.html` there's a dublicate `- {% endif %}` from the previous unwanted change to remove.

**Review by sylhare — CHANGES_REQUESTED** [review]
Also some icons might have been missed as they are not rendering for me. 

For example `far fa-file-text`:
image

Also, the share icons seems to still have `fa` like the facebook one `fa fa-facebook-square fa-2x` causing them I suppose to not be rendered.
image

File: _includes/post_info.html (hunk 1 above)
Comment (lbcnz) [reply _includes/post_info.html:10]: Sorry.

**Comment by lbcnz** [comment]
Fixed those.

Screenshot from 2021-02-18 15-24-28

Screenshot from 2021-02-18 15-24-41

**Comment by lbcnz** [comment]
For documentation purposes.

+ [FontAwesome 5 Icons](https://fontawesome.com/icons?d=gallery)
+ [FontAwesome 4 Icons](https://fontawesome.com/v4.7.0/icons/)

**Review by sylhare — APPROVED** [review]

**Comment by sylhare** [comment]
Nice thanks 👌

**Pull request merged**